File: js/romanNumerals.js
exports.toRoman = function(num) {
    let output = ""
    const romanNumeralObj = {
        M : 1000,
        D : 500,
        C : 100,
        L : 50,
        X : 10,
        V : 5,
        I : 1,
    }
   
    let romanArr = Object.entries(romanNumeralObj)
    let tempNum = num
    let romanAnsw = []
    // for(let i = 0; i < romanArr.length; i++){
    //     if (tempNum % romanArr[i][1] !== 0){
    //         romanAnsw.push(romanArr[i])
    //         tempNum = tempNum - (romanArr[i][1])
    //     } 
    //     console.log(tempNum)
        
    // }
    console.log(parseInt(romanArr[0][1]))
    return romanAnsw
};
exports.toRoman()

File: python/roman_numerals.py
def to_roman(num):
    output = ""
    roman_Num = {
        'M' : 1000,
        'CM' : 900,
        'D' : 500,
        'CD' : 400,
        'C' : 100,
        'XC': 90,
        'L' : 50,
        'XL' :40,
        'X' : 10,
        'IX' : 9,
        'V' : 5,
        'IV' : 4,
        'I' : 1,
    }

    for key in roman_Num:
        while num/roman_Num[key] >= 1:
            output += key
            num -= roman_Num[key]
    return output